backend/src/backend/logging.clj

```diff
@@ -27,12 +27,22 @@
   ([from to]
    (j/query db [(str "SELECT * FROM system_log WHERE DATE(created) between '" from "' and '" to "'")])))
 
+#_(defn set-device-id
```

Comment:
Because it is not currently being used, however I think it will be useful once we resolve the device_id-issue. I should probably have figured that out prior to creating the pull request to avoid commenting out, will look into it now.